includes/blocks/class-sensei-course-progress-block.php

```diff
@@ -93,11 +93,11 @@ public function render_course_progress( $attributes ) : string {
 
 		return '
 			<div ' . Sensei_Block_Helpers::render_style_attributes( $class_names, $text_css ) . '>
-				<section class="wp-block-sensei-lms-progress-heading sensei-course-progress__heading">
-					<div class="wp-block-sensei-lms-progress-heading__lessons sensei-course-progress__lessons">' . $lessons_text . '</div>
-					<div class="wp-block-sensei-lms-progress-heading__completed sensei-course-progress__completed">' . $completed_text . '</div>
+				<section class="wp-block-sensei-lms-progress-heading sensei-progress-bar__heading">
+					<div class="wp-block-sensei-lms-progress-heading__lessons sensei-progress-bar__lessons">' . $lessons_text . '</div>
+					<div class="wp-block-sensei-lms-progress-heading__completed sensei-progress-bar__completed">' . $completed_text . '</div>
 				</section>
-				<div role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100" ' . Sensei_Block_Helpers::render_style_attributes( [ 'wp-block-sensei-lms-progress-bar', 'sensei-course-progress__bar' ], $bar_background_css ) . '>
+				<div role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100" ' . Sensei_Block_Helpers::render_style_attributes( [ 'wp-block-sensei-lms-course-progress', 'sensei-progress-bar__bar' ], $bar_background_css ) . '>
```

Comment:
I'm wondering if `wp-block-sensei-lms-progress-bar` should have been renamed here. Currently, there's a mismatch of the CSS class names on this element in the editor vs. on the front-end. Ideally, these should match in both places.

Reply:
Oh now I didn't want to change it at all, thanks, will redo the change

Reply:
Actually, I'm not sure that ```wp-block-sensei-lms-progress-bar``` class exists anymore. I can't find it anywhere.  When this class was first use there was scss file with ```.wp-block-sensei-lms-progress-bar``` but now I don't see . However the class wp-block-sensei-lms-course-progress does exist. 

Btw I tried removing this class from here all together and everything worked.

Reply:
OK, if you want to remove it I'm OK with that.

Reply:
@donnapep so I took a different approach and replaced remaining occurrences of wp-block-sensei-lms-progress-bar with wp-block-sensei-lms-course-progress 

https://github.com/Automattic/sensei/pull/4572/commits/04385822c383af36a5a70bb0f7b854d8e209aecf#diff-ad41cce68f3f19eb5eb9005f5ae4ddd1202bfd05af0d39d84138f1fb90ecf030R70